<launch>
    <include file="$(find realsense2_camera)/launch/rs_camera.launch"/>
    <node pkg="motpy_ros" name="face_detect" type="face_detect.py" output="screen">
        <param name="weights_url" value="https://github.com/opencv/opencv_3rdparty/raw/dnn_samples_face_detector_20170830/res10_300x300_ssd_iter_140000.caffemodel" />
        <param name="config_url" value="https://raw.githubusercontent.com/opencv/opencv/master/samples/dnn/face_detector/deploy.prototxt" />
        <!-- <param name="weights_path" value="/home/ubuntu/.cache/motpy_ros/face_tracking/opencv_face_detector.caffemodel"/>
        <param name="config_path"  value="/home/ubuntu/.cache/motpy_ros/face_tracking/deploy.prototxt"/> -->
        <param name="tracking_size/width" value="640" />
        <param name="tracking_size/height" value="360" />
    </node>
</launch>  